### PR TITLE
Fixed several inconsistencies and compile errors.

### PR DESCRIPTION
**Pull request by brminich**
Test takes transport name as an input parameter. Latencies checking is
removed.

### CODE REVIEW

File: test/examples/active_message.c [hunk 1]
```diff
@@ -11,304 +11,258 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-static int rank;
+#define CHKERR_JUMP(cond, msg, label)           \
+    do {                                        \
+        if (cond) {                             \
+        fprintf(stderr, "Failed to %s\n", msg); \
```
Comment (yosefe) [inline test/examples/active_message.c]: ןמגקמאשאןםמ

File: test/examples/active_message.c [hunk 2]
```diff
@@ -11,304 +11,258 @@
...
+    uint8_t id = 0;
+    void *arg;
+    char *tl_name;
+    struct iface_info if_info;
+    int exit_fail = 1;
+
+    if (argc > 1) {
+        tl_name = argv[1];
+    } else {
+        tl_name = "rc";
```
Comment (yosefe) [inline test/examples/active_message.c]: i think there should not be a default value - just exit with error if tl was not specified

File: test/examples/active_message.c [hunk 3]
```diff
@@ -11,304 +11,258 @@
...
+        status = uct_pd_open(pd_resources[i].pd_name, pd_config, &iface_p->pd);
+        uct_config_release(pd_config);
+        CHKERR_JUMP(UCS_OK != status, "open protected domains", release_pd);
+
+        status = uct_pd_query_tl_resources(iface_p->pd, &tl_resources, &num_tl_resources);
+        CHKERR_JUMP(UCS_OK != status, "query transport resources", close_pd);
+
+        /* Go through each available transport and find the proper name */
+        for (j = 0; j < num_tl_resources; ++j) {
+            if (0 == strcmp(tl_name, tl_resources[j].tl_name)) {
```
Comment (yosefe) [inline test/examples/active_message.c]: lets also pass the device name from command line

**Comment by yosefe** [comment]
@brminich please also add this test to jenkins script in contrib/test_jenkins.sh

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/894/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1505/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/914/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1523/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1525/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/917/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/919/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1527/ for details.

**Comment by miked-mellanox** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1531/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/925/ for details (Mellanox internal link).

File: test/examples/Makefile.am [hunk 4]
```diff
@@ -17,5 +17,10 @@ installcheck-local:
 	$(CC) -o ucp_hello_world $(examplesdir)/ucp_hello_world.c -lucp -I$(includedir) -L$(libdir)
 if HAVE_MPI
 	$(MPICC) -o active_message $(examplesdir)/active_message.c -luct -lucs -I$(includedir) -L$(libdir)
+	@for dev in $$(ibstat -l); do \
+	    hca="$${dev}:1"; \
+	    echo "Running active_message example on $$hca"; \
+	    mpirun -np 2 -mca pml ob1 -mca btl sm,self ./active_message $$hca "ud"; \
```
Comment (yosefe) [inline test/examples/Makefile.am]: lets not run tests as part of "make distcheck"

File: contrib/test_jenkins.sh [hunk 5]
```diff
@@ -70,13 +70,18 @@ if [ -n "$JENKINS_RUN_TESTS" ]; then
...
 
     # todo: check in -devel mode as well
     ../contrib/configure-release --with-mpi --prefix=$ucx_inst
     make $make_opt install
+    
+    # Prevent out tests from using installed UCX libraries
+    export LD_LIBRARY_PATH=${ucx_inst}/lib:$LD_LIBRARY_PATH
+
+    # check examples
+    make -C test/examples installcheck-local
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: can you please put "mpirun" here instead of inside the makefile?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1551/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/944/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1559/ for details.

**Comment by yosefe** [comment]
@brminich can you please squash all commits into one?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1563/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/956/ for details (Mellanox internal link).

**Pull request closed without merging**